sub/source/docs/sub-hardware.rst

```diff
@@ -0,0 +1,83 @@
+.. _sub-hardware:
+
+==================================
+Default Sub Hardware Configuration
+==================================
+
+While the Sub firmware is currently being generalized to allow more flexibility, such as RC control, re-arrangement of outputs and inputs, etc.,
+current code is targeted for a specific ROV configuration with the following hardware setup:
+
+- Control is via a surface tether from a ground control station via MAVLink commands to the Sub's autopilot telemetry port, usually employing a joystick and multiple buttons for mode changes and peripheral hardware control. Control functions and autopilot outputs for thrusters and peripheral hardware have specific defaults (see :ref:`Sub defaults <sub-defaults>` below)
+- A depth sensor (underwater barometer) is required.
+- Frame configuration is defaulted to Vectored with 6 thrusters.
+
+.. _sub-defaults:
+
+Sub Defaults
+============
+
+* Default :ref:`frame type <FRAME_TYPE>` is VECTORED
+* RC is disabled via :ref:`RC_PROTOCOLS<RC_PROTOCOLS>` = 0 and :ref:`FLTMODE_CH<FLTMODE_CH>` = 0.
```

Comment:
```suggestion
* Radio control is disabled via :ref:`RC_PROTOCOLS<RC_PROTOCOLS>` = 0 and :ref:`FLTMODE_CH<FLTMODE_CH>` = 0. Water absorbs radio waves, so RC is generally impractical unless the vehicle is at the surface, or the signal is converted to some other medium in between (e.g. using a tether cable, or acoustic modems).
```

- People starting out with a Sub vehicle don't necessarily know what RC is.
- People gaining first exposure to underwater vehicles may not be aware of its electromagnetic absorption spectrum
    - Longer term I'm considering including a broader "challenges of the underwater environment" section to explain things like this, which this could link to, but it seems alright to include a relevant note here for now.

Reply:
I like it